### DISCUSSION TITLE
臺灣版GPT-4?

### DISCUSSION THREAD

**Discussion by Eileen050824** [1101-GS4719-C/Python-Community-GS4719-C#1926] [科技新知]
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/160737635/b9677841-49f7-481c-bf5d-c8a3ca9e8e96)
聯發創新基地昨日(3/7)下午宣布，開源釋出MediaTek Research Breeze-7B大型語言模型供大眾使用，並表示此模型能夠精準理解和生成中、英兩種語言，希望能促進AI發展。
我自己認為這是個不錯的推動，畢竟近期全球AI領域研究蓬勃發展，但多數研究與開發項目都以英文的大型語言模型進行。如果能夠連結學術界專家和應用開發者，一起為推進繁體中文AI的創新與應用而努力，不僅為即時翻譯、智慧客服對話等顯著的效能提升，也相信這次Breeze-7B繁體中文大型語言模型將有益於學術界與業界在AI的發展。
不知道大家對這個模型的推出有什麼想法？
來源：風傳媒

**Comment by KH1126** [comment]
我認為聯發科釋出語言模型是一個極具潛力的作法。首先，這個模型不僅能夠處理中文，還能夠處理英文，這對於中文語境下的AI應用是一大進步。過去，大多數的大型語言模型都是以英文為主，對於中文處理的能力有限，因此這個中英文兼容的模型將為中文語境下的AI應用提供更多可能性。

此外，通過開源還可以增加debug機會。

**Comment by wanderer657** [comment]
我發現聯發創新基地昨天公開了MediaTek Research Breeze-7B大型語言模型，我覺得這對AI領域是個不錯的推動。其他AI工具大多只支援英文，對中文使用者不太友善。有了這個繁體中文模型，使用起來會更順暢。我自己在使用英文AI工具時常感到困擾，但這個模型的出現給了我希望。期待它的廣泛應用！

**Comment by 20050201Shawn** [comment]
我認為這個模型對繁體中文使用者蠻友善的。chatgpt大多吃的是簡體中文資料，用語比較符合對岸的習慣，有了這個模型，英文能精準的和繁體中文做切換，台灣人使用體驗更佳。此外，因為大多語言模型是以英語進行，此中英兼容的模型能透過英文間接和其他國語言進行翻譯，促進多國語言和繁體中文間的交流。
